src/mod33.py

```diff
@@ -0,0 +1,30 @@
+"""Homework1 - 33 lines"""
+import random
```

Comment:
here you have import after task description without blank lines. earlier you split it with 2 blank lines. Styling should be consistent